modules/terrain/src/lib/helpers/skirt.js

```diff
@@ -0,0 +1,133 @@
+import {concatenateTypedArrays} from '@loaders.gl/loader-utils';
+
+/**
+ * Add skirt to existing mesh
+ * @param {object} attributes - POSITION and TEXCOOD_0 attributes data
+ * @param {any} triangles - indices array of the mesh geometry
+ * @param {number} skirtHeight - height of the skirt geometry
+ * @returns - geometry data with added skirt
+ */
+export function addSkirt(attributes, triangles, skirtHeight) {
+  const outsideEdges = getOutsideEdges(triangles);
```

Comment:
This should be an optional argument to the `addSkirt` function. The quantized mesh metadata already contains this information, so you don't need to compute it. (Martini and Delatin output from the `TerrainLoader` doesn't compute this automatically)

Reply:
Where can I take relevant quantized dataset to test it?

Reply:
Maybe simplest to take either of the `.terrain` files in the test suite: https://github.com/visgl/loaders.gl/tree/master/modules/terrain/test/data

Alternatively if you want to test with multiple neighboring tiles in a region, you can generate some from Python. Would it be helpful for me to generate a handful of tiles for you, and I can share a zip file here?

Reply:
I'll use test on for now.